### DISCUSSION TITLE
How to hide Jupyter Notebook language

### DISCUSSION THREAD

**Discussion by admcurry** [anuraghazra/github-readme-stats#2581] [Q&A] [answered]
I'm trying to hide the Jupyter Notebook language, and I managed to hide HTML by adding `&hide=html` to the end of the link, but I'm not sure what name I would have to use to hide Jupyter. Any ideas?

**Comment by kahn-jms** [comment, marked as answer]
Hi, how to do this is mentioned in the project's [README](https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options), under the language card's exclusive options:

> Warning Language names should be URI-escaped, as specified in [Percent Encoding](https://en.wikipedia.org/wiki/Percent-encoding) (i.e: `c++` should become `c%2B%2B`, `jupyter notebook` should become `jupyter%20notebook`, etc.) You can use [urlencoder.org](https://www.urlencoder.org/) to help you do this automatically.

That being said, a list of all the language names or reference where they can be looked up would be great.

**Reply by Zo-Bro-23** [reply]
The list is available as JSON [here](https://github.com/anuraghazra/github-readme-stats/blob/master/src/common/languageColors.json). Feel free to create a PR to add this list in markdown format. Since the JSON is automatically created though, you will have to edit `/scripts/generate-langs-json.js` too. I will approve the PR if you create one!

**Reply by admcurry** [reply]
Thank you!
